docs/reference/tools-reference/python-tool.md

```diff
@@ -12,6 +12,25 @@ Users can effortlessly create Python tools, edit code, and verify results with e
 | Inputs | -      | List of tool function parameters and its assignments | -       |
 
 
+### Types
+
+| Type | Python annotation             | Description                                |
+| ---- |-------------------------------|--------------------------------------------|
+| int | param: int                    | Integer type                               |
+| bool | param: bool                   | Boolean type                               |
+| string | param: str                    | String type                                |
+| double | param: float                  | Double type                                |
+| list | param: list or param: List[T] | List type                                  |
+| object | param: dict or param: Dict[K, V] | Object type                                |
+| xxConnection | param: xxConnection | Connection type, will be handled specially |
```

Comment:
xxConnection is hard to understand. May just put Connection here. and give a link to connection concetp